src/js/Item.ts

```diff
@@ -102,7 +102,7 @@ export class Item<Model extends ModelAttributes> {
         }
 
         const element = this.document.createElement('div') as HTMLElement;
-        let image: HTMLElement = this.document.createElement('div');
+        let image: HTMLElement = this.document.createElement('a');
```

Comment:
photoswipe recommended gallery markup  
(https://photoswipe.com/getting-started/#required-html-markup)  
` Each element that matches the selector should be or should contain link <a> element`  

actually, I think it may no longer be required as I later changed to itemData filter as PS images source (https://photoswipe.com/data-sources/#dynamically-generated-data) so it's possibly safe to remove - I think it'd cleaner to keep it as PS suggests though